src/com/sap/piper/DownloadCacheUtils.groovy

```diff
@@ -47,6 +49,14 @@ class DownloadCacheUtils {
             return false
         }
 
+        // Do not enable the DL-cache when a sidecar image is specified.
+        // This is necessary because it is currently not possible to connect a container to multiple networks.
+        // Can be removed when docker plugin supports multiple networks and jenkins-library implemented that feature
+        if (script.env.SIDECAR_IMAGE) {
+            script.echo "D/L cache disabled while running with sidecar image (${script.env.SIDECAR_IMAGE})"
```

Comment:
do we have an issue to track this? maybe we should create one and put a link to it here for transparency reasons, also users might be confused about the reason for this limitation.

Reply:
I have not yet fully understood whether this is really a limitation of the docker Jenkins plugin, or our own dockerExecute step. In either case though, creating an issue in *this* repo makes sense to track it, no?

Regarding the user confusion, you mean to put more information into the log message, correct?

Reply:
> creating an issue in this repo makes sense to track it

I think it makes sense regardless of where it can be fixed because it is a known limitation of this software.

Reply:
Done.